examples/auth/toolpad/application.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: application
+spec:
+  {
+    authentication:
+      { providers: [{ provider: github }, { provider: google }], restrictedDomains: [] },
```

Comment:
🤔 Why does this file use `{` and `}` everywhere?

Reply:
I probably changed this manually and not really used to writing in YAML :D, I should be able to change it in the UI now.